.github/workflows/linkchecker.yml

```
@@ -4,11 +4,19 @@ jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
     steps:
+    - name: Wait for Vercel preview deployment to be ready
+      uses: patrickedqvist/wait-for-vercel-preview@master
+      id: waitForVercelPreviewDeployment
+      with:
+       token: ${{ secrets.GITHUB_TOKEN }}
+       max_timeout: 180
```

QQ: are these seconds ?

---

Yes: https://github.com/patrickedqvist/wait-for-vercel-preview#max_timeout

Then just for tips:
```suggestion
       max_timeout: 180 #seconds
```